apps/builder/app/builder/features/style-panel/style-source/style-source.tsx

```diff
@@ -327,6 +329,7 @@ const StyledSourceButton = styled(Box, {
   display: "inline-flex",
   borderRadius: theme.borderRadius[3],
   padding: theme.spacing[4],
+  minWidth: 32,
```

Comment:
can use theme.spacing

Reply:
scaling is hard to understand `13: "32px"`